Assets/Scripts/PlayerLevelEditor.cs

```diff
@@ -33,37 +44,45 @@ private void Update()
     /// </summary>
     private void PlaceTile()
     {
-        Color cursorColor = cursor.GetComponent<SpriteRenderer>().color; //cursor's opacity
-        Color tileColor; //cursor's opacity
+        Color invalidColor; //invalid sprites opacity/color
+        Color tileColor; //current tile's opacity/color
         Ray ray = Camera.main.ScreenPointToRay(Input.mousePosition);
         RaycastHit2D rayHit = Physics2D.Raycast(ray.origin, ray.direction, Mathf.Infinity, allTilesLayer);
+         
 
         if (rayHit.collider != null)//checks whether cursor is over an already placed tile (includes ground, walls, player spawned tiles and foreground items)
         {
+            invalidColor = new Color(1, 1, 1, 1);
             tileColor = new Color(1, 0, 0, 0.5f);
```

Comment:
Instead of creating new colors in code, if you serialize a Color field at the top of a class it will give you a color picker you can customize in the inspector